src/server/service/achievements.ts

```diff
@@ -28,7 +28,7 @@ const doubleAgentAchievement: Achievement = {
 
 const busterStreakAchievement: Achievement = {
   name: "Buster Streak",
-  description: "Bust three bots in a row",
+  description: "Successfully bust all bots in (3) consecutive games",
```

Comment:
```suggestion
  description: "Successfully bust all bots in 3 consecutive games",
```
unless it's official copy